feign-demo/src/test/resources/application-override.yml

```diff
@@ -5,6 +5,13 @@ logging:
     cucumber: TRACE
     com.nortal.test: INFO
     org.springframework: INFO
+    com.nortal.test.feign.demo.api: DEBUG
+
+feign:
+  client:
+    config:
+      default:
+        loggerLevel: full
```

Comment:
Looks like this is not needed if its specified from bean config?

Reply:
this is required if we want to change the client logging level configuration, I can add a comment explaining this if needed